kb-importer/src/main/java/org/eclipse/steady/kb/task/ImportVulnerability.java

```diff
@@ -56,6 +56,7 @@
   private static final String OVERWRITE_OPTION = "o";
   private static final String DIRECTORY_OPTION = "d";
   private static final String VERBOSE_OPTION = "v";
+  private static final String DELETE = "del";
```

Comment:
@pedrogalvao I know that all those identical `private static final String`s already existed in both the command `Import.java` and the two tasks, but don't you think it would be better to declare them as `public static final` in the command, and reference them via `Import.DELETE` etc. in the tasks?

Reply:
Yes, I agree